back/engines/commercial/user_custom_fields/spec/acceptance/stats_users_spec.rb

```diff
@@ -630,4 +638,128 @@ def xlsx_worksheet_to_array(worksheet)
       end
     end
   end
+
+  describe 'by_age' do
+    let(:start_at) { Time.zone.local(2019) }
+    let(:end_at) { Time.zone.local(2020).end_of_year }
+
+    # I made my life easier here, at the expense of test coverage, by using a group
```

Comment:
Please do not write in the "I" form, because the reader does not know who "I" is.